projenrc/customized-license.ts

```diff
@@ -0,0 +1,19 @@
+import { IResolver, License } from "projen";
+import { TypeScriptProject } from "projen/lib/typescript";
+
+const SPDX = "Apache-2.0";
```

Comment:
I suspect it has been this license previously, just wanted to note this as we're using the MPL elsewhere (I don't know the differences between those two and whether they were picked on purpose even 😅).

Reply:
Yeah, I pointed that out in the commit message as well. I'm planning to merge this and then open a new PR updating it to MPL but I'll wait until Daniel's back from vacation to comment since I wanted to make sure the usage of Apache wasn't intentional here. (I suspect it was just an accidental oversight since it's the default that Projen uses.)